flutter/test/load_image_list_test.dart

```diff
@@ -0,0 +1,176 @@
+@TestOn('vm')
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+import 'mocks.dart';
+import 'sentry_flutter_test.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  late Fixture fixture;
+
+  tearDown(() {
+    fixture.channel.setMockMethodCallHandler(null);
+  });
+
+  for (var platform in [
+    MockPlatform.android(),
+    MockPlatform.iOs(),
+    MockPlatform.macOs()
+  ]) {
```

Comment:
Basically the same as the previous `load_android_...` but runs for three platforms